platform/admin/docs/src/main/resources/ManContents.adoc

```diff
@@ -1,99 +1,84 @@
+|`ddf.ui.admin.api`
+|List of Configuration Filters
+|===
 
-== Verify
+==== Defining Platform Settings
 
-. Verify the appropriate features for the DDF application have been installed using the
-`features:list` command to view the KAR file's features.
+The platform settings can be configured by the `<INSTALL_HOME>/etc/system.properties` file.
 
-. Verify that the bundles within the installed features are in an active state.
+The settings in this file can be changed at any time during the {branding} lifecycle (before installation, after, or during run-time) and will take effect after a system restart.
 
-== Uninstall Applications
-[WARNING]
-====
-It is very important to save the KAR file or the feature repository URL for the application prior to an uninstall so that the uninstall can be reverted if necessary.
-====
+=== Console Commands
 
-If the DDF application is deployed on the DDF Kernel in a custom installation (or the application has been upgraded previously), i.e., its KAR file is in `<INSTALL_DIRECTORY>/deploy` the directory, uninstall it by deleting this KAR file.
+The application service comes with various console commands that can be executed on the {branding} system console.
 
-Otherwise, if the DDF application is running as part of the DDF distribution zip, it is uninstalled *the first time and only the first time* using the `features:removeurl` command:
-
-.Uninstall DDF application from DDF distribution
-----
-features:removeurl -u <DDF application's feature repository URL>
-
-Example:   features:removeurl -u mvn:ddf.catalog/catalog-app/2.3.0/xml/features
-----
+==== Application Service Interfaces
 
-The uninstall of the application can be verified by the absence of any of the DDF application's features in the `features:list` command output.
+The Application service has multiple ways of interacting with it.
+These methods include operations that can be performed by integrators, administrators, and end users.
 
-=== Revert the Uninstall
+==== Installing and Uninstalling
 
-If the uninstall of the DDF application needs to be reverted, this is accomplished by either:
+The Admin App installs this service by default.
+It is recommended to NOT uninstall the application service unless absolutely necessary.
 
-* copying the application's KAR file previously in the `<INSTALL_DIRECTORY>/deploy` directory, OR 
+==== Configuring
 
-* adding the application's feature repository back into DDF and installing its main feature, which typically is of the form `<applicationName>-app`, e.g., `catalog-app`.
+None.
 
-.Reverting {branding} application's uninstall
-----
-features:addurl <DDF application's feature repository URL>
-features:install <DDF application's main feature>
-
-Example:
-    ddf@local>features:addurl mvn:ddf.catalog/catalog-app/2.3.0/xml/features
-
-    ddf@local>features:install catalog-app
-----
+=== Administrative User Interface
 
-== Upgrade 
+The Admin UI is the centralized location for administering the system.
```

Comment:
It probably wouldn't hurt to also provide the default address to where the Admin Console is (https://localhost:8993/admin)